Updated README file with the latest instructions. (#21)
OS1-logistics/vehicle-reference-app: README.md

2. Open a Nix Shell and execute the following commands:

```
TENANT_NAME=<Add your tenant name here>
TENANT_URL=<Add your tenant url here>
```
3. Update your `/etc/hosts` file:

#### Linux

```
echo "127.0.0.1 ${TENANT_NAME}-cdev.${TENANT_URL}" >> /etc/hosts
```

#### Mac

```
echo "127.0.0.1 ${TENANT_NAME}-cdev.${TENANT_URL}" | sudo tee -a /etc/hosts
```

#### Windows

Windows stores the `hosts` file at `C:\Windows\System32\drivers\etc\`. To modify the file:

1. Open a Notepad as an Administrator.
2. In the notepad, go to **File > Open** and navigate to `C:\Windows\System32\drivers\etc\`.
3. The `hosts` file won’t be visible by default because it doesn’t have a file extension. To make it visible, change the file type filter from **Text Documents (.txt)** to **All File (.*)**, and then select the `hosts` file. 
4. After opening the file, you can add your energy at the end of the file, following the format in the above examples for Linux and Mac.

```
127.0.0.1 ${TENANT_NAME}-cdev.${TENANT_URL}
```

> These commands append your tenant’s domain to the `etc`

Author: Removed.

#### **Running Locally**

After configuring your proxy, follow the steps below to run the application on your local machine:

1. Install the necessary dependencies

```
yarn install
```

2. Next, start your local server. 

```
yarn start
```

## **How to setup the Backend (Server)**

> The backend server facilitates vehicle creation, update, and lifecycle management. It also provides a baseline reference on how to model a vehicle or any other participant in a logistics network such as Facility using the Participant Service. The intention of this is to give the developer an easy point of reference for a few things:

Reviewer: I think it would be good to update this to talk to the 3rd party developer more than about them.

```suggestion
The backend server facilitates vehicle creation, update, and lifecycle management. It also provides a baseline reference on how to model a vehicle or any other participant in a logistics network such as a Facility using the Participant Service. In particular this demonstrates the following:
```

* Generating access token
* Communicating with Participant Service

### **Setup**

1. Install dependencies: `yarn`.
2. Rename your `.env.local.skeleton` to `.env.local`.
3. Add values for each required variable in the `.env.local` file. These details are available on your app summary page in the Developer Portal. Refer to the [Getting Started with the OS1 Platform](https://docs.getos1.com/docs/getting-started-1#step-5-retrieving-and-using-your-platform-credentials) guide for obtaining these details.  

### **Running the App**

1. Start your application: `yarn start`.

> 2. The `Vehicle's` participant type and vehicle attributes will be defined during the application startup if they don't exist.

Reviewer: Might be good to mention that a browser will open to http://localhost:4500. However they will probably need to update the url to be `https://${TENANT_NAME}-cdev.${TENANT_URL}:4500` in order for the Keycloak redirection to work. This was the case in the original version and I assume it's the same.

> 2. The `Vehicle's` participant type and vehicle attributes will be defined during the application startup if they don't exist.

Reviewer: This line about the participant and vehicle attributes being defined during startup belongs in the server app section.
